src/main/python/ecir2019_axiomatic/effectiveness.py

```diff
@@ -0,0 +1,94 @@
+# -*- coding: utf-8 -*-
+"""
+Anserini: A toolkit for reproducible information retrieval research built on Lucene
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+import os
+import json
+import ast
+from operator import itemgetter
+from inspect import currentframe, getframeinfo
+import logging
+
+logging.basicConfig()
+class Effectiveness(object):
+    """
+    Handle the performace. For example, get all the effectiveness of one method(has multiple parameters).
```

Comment:
According to PEP 8, the summary should be on the line with `"""`
https://www.python.org/dev/peps/pep-0008/